### ISSUE TITLE
Content Items in Nuget 3

### ISSUE THREAD

**Issue by ToddThomson** [NuGet/Home#688]
Is it possible to add content items to a nuget 3 package? Specifically, I would like to add css and cshtml files along with the lib dll ( essentially a MVC6 component ).

**Comment by emgarten** [comment]
Content files can be placed under the content folder inside the nupkg.

See the docs for more details:
https://docs.nuget.org/create/creating-and-publishing-a-package

**Comment by yishaigalatzer** [comment]
Correction: Content items are currently not supported in ASPNET5/MVC6 apps, this has nothing to do with NuGet 3, it has to do with the aspnet project system not supporting content items in NuGet packages and taking content through Bower. This is probably a good discussion to have on github.com/aspnet/home

**Comment by ToddThomson** [comment]
@yishaigalatzer Thank-you. I found that the dnx pack and publish using --no-source would not support creating content items. I really just wanted to know if this was dnx specific or was for all nuget 3 packages.
To confirm: You are saying that with nuget 3 tools we will be able to support "content" within packages.
Also, how will nuget content items work with vs2015 - if I have an asp.net 5 webapplication then will it utilize dnx?
Sorry for all the questions! I think I jumped the gun a bit with thinking vs2015 RC would have much of the documentation done!
